backend/battle/views.py

```diff
@@ -69,3 +69,20 @@ def get_queryset(self):
 class BattleDetail(DetailView):
     model = Battle
     template_name = "battle/battle_detail.html"
+
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()  # pylint: disable=attribute-defined-outside-init
+        context = self.get_context_data(object=self.object)
+        return self.render_to_response(context)
```

Comment:
Why do you need to override this `def get`?

Reply:
It's really misleading, thank you :)